lsp-mode.el

```diff
@@ -2967,35 +2949,54 @@ SLEEP-INTERVAL is the sleep interval between each retry."
         (file-error (setq success t))))
     port))
 
-(defun lsp-tcp-connection (command-fn)
+(defun lsp-tcp-connection (command)
```

Comment:
Sorry, but why was this renamed to `command`

Reply:
It looks like a leftover change, will be fixed.

Reply:
Fixed with 03e727718e6da93a2344392545534102de715c5c